.github/workflows/ci.yml

```diff
@@ -180,17 +188,40 @@ jobs:
       run: |
         choco install nuget.commandline
         nuget add -source local nuget/ParquetSharp.${{ steps.get-version.outputs.version }}.nupkg
+    - name: Setup .NET variables
+      id: dotnet
+      shell: bash
+      run: |
+        major=${{ matrix.dotnet }}
+        case $major in
+          3)
+            framework=netcoreapp3.1
+            ;;
+          4)
+            framework=net461
+            # We build for .NET framework with .NET Core 3.1 SDK
+            major=3
+            ;;
+          5)
+            framework=net5.0
+            options='-p:Channel=experimental'
```

Comment:
I would not have thought that net5 was still considered experimental since it had now been released. Is this option really necessary?

Reply:
I have added that flag in the project file to avoid forcing users to install .NET 5.0 to build and run the tests. Happy to remove it if we can assume every contributors will have it installed.

Reply:
Happy to keep it, but then needs a comment to explain why it's there :-)

Reply:
Makes sense, will add it once I get the chance 👌

Reply:
I figured out a way to automatically select the target frameworks based on runtime availability. See commit 5d35c21d196763905d6a4dc9814241124268a1b3.

Reply:
As discussed offline, I changed it once more to statically include .NET 5.0 as a target framework for the tests.